inventory/def.cf

```diff
@@ -0,0 +1,26 @@
+bundle agent inventory_def_file_control
+{
+  vars:
+  # Each input should be guarded with both an enable and !disable class so that
```

Comment:
If it's enabled by default, how should someone who does not need or want it disable it? Cancelling a class is not straightforward.

I can cite [this conversation on the help list](https://groups.google.com/d/msg/help-cfengine/aEjW_IqaFKw/S1T99ElDC4sJ) where at least I convinced @neilhwatson